draft-ietf-quic-qpack.md

```diff
@@ -1339,7 +1341,7 @@ An attacker can try to cause an endpoint to exhaust its memory. QPACK is
 designed to limit both the peak and stable amounts of memory allocated by an
 endpoint.
 
-The amount of memory used by the encoder is limited by the protocol using
+The amount of memory used by the decoder is limited by the protocol using
 QPACK through the definition of the maximum size of the dynamic table, and the
 maximum number of blocking streams. In HTTP/3, these values are controlled by
```

Comment:
I think this is really dependent on the sense of "used by."  The protocol limits the amount of memory the encoder can cause to be consumed at the decoder.  Maybe something like....

```suggestion
QPACK uses the definition of the maximum size of the dynamic table and the
maximum number of blocking streams to limit the amount of memory the
encoder can cause the decode to consume. In HTTP/3, these values are
controlled by
```